### PR TITLE
MOTOR-415 Remove quoted text/code for __aenter__ and __aexit__ helpers

### PR DESCRIPTION
**Pull request by prashantmital**
MOTOR-416 Remove quoted text/code for __aiter__ and __anext__ helpers

### CODE REVIEW

File: motor/motor_gridfs.py [hunk 1]
```diff
@@ -368,15 +363,11 @@ def __init__(self, root_collection, delegate=None, session=None,
                 disable_md5=disable_md5,
                 **kwargs)
 
-    if PY35:
-        # Support "async with bucket.open_upload_stream() as f:"
```
Comment (ShaneHarvey) [inline motor/motor_gridfs.py:372]: Can we keep this comment?

File: motor/docstrings.py [hunk 2]
```diff
@@ -1189,89 +1132,4 @@ async def coro():
...
-    transaction. If ``None`` (the default) the :attr:`read_preference`
-    of this :class:`Database` is used. See
-    :mod:`~pymongo.read_preferences` for options.
-
-:Returns:
-  The return value of the ``coro``.
-
-.. versionadded:: 3.9
-"""
+"""
```
Comment (ShaneHarvey) [inline motor/docstrings.py:1135]: Newline.

File: motor/docstrings.py [hunk 3]
```diff
@@ -1122,7 +1066,6 @@ async def add_3_to_x():
     key, if not given :data:`~pymongo.ASCENDING` is assumed
 """
 
-
```
Comment (ShaneHarvey) [inline motor/docstrings.py:1125]: Are these blank space changes intentional?

File: motor/motor_gridfs.py (hunk 1 above)
Comment (prashantmital) [reply motor/motor_gridfs.py:372]: Done.

File: motor/docstrings.py (hunk 3 above)
Comment (prashantmital) [reply motor/docstrings.py:1125]: Yes. Some had 2 empty lines and others 1, I changed them all to 1.

File: motor/docstrings.py (hunk 2 above)
Comment (prashantmital) [reply motor/docstrings.py]: Done.

**Pull request merged**